content/go/concepts/working-with-json/working-with-json.md

```diff
@@ -0,0 +1,228 @@
+---
+Title: 'Working with JSON'
+Description: 'Working with JSON files, requests, or responses is an essential part of interacting with data that will be used to exchange information between the Frontend and Backend in web applications.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'JSON'
+  - 'Marshal'
+  - 'Unmarshal'
+  - 'Encode'
+  - 'Decode'
```

Comment:
These are not valid tags, please check [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) for more reference. You can use tags from `tags.md` as follows:

```suggestion
  - 'JSON'
  - 'Data'
  - 'Development'
  - 'Objects'
```